zzk/service/lock_test.go

```diff
@@ -118,7 +118,24 @@ func (t *ZZKTest) TestEnsureServiceLock(c *C) {
 
 	// Release lock
 	close(exit)
-	runtime.Gosched()
+
```

Comment:
Sanity check - Is this necessary?

Just asking because it seems unrelated to the other changes.